src/pages/settings/Profile/PronounsPage.js

```diff
@@ -106,51 +66,76 @@ class PronounsPage extends Component {
             })
             .sortBy((pronoun) => pronoun.text.toLowerCase())
             .value();
-    }
+
+        setPronounsList(pronouns);
+    }, [props]);
```

Comment:
@aimane-chnaif You mean props.currentUserPersonalDetails has changed, right? Is `pronouns` ever null/undefined? Because if it is then we cannot use that in the deps array.